### DISCUSSION TITLE
Add DivestOS apps to the Accrescent App Store.

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#186] [Ideas]
Add DivestOS apps (Mull, Mulch, Hypatia) to the Accrescent app store. 

Accrescent has better security approaches compared to F-Droid.

**Comment by rsssxd** [comment]
![Screenshot_20230128-144510_Accrescent](https://user-images.githubusercontent.com/60405617/215264735-23a1a37b-b197-429c-b7be-d5982d4d49df.png)

**Comment by SkewedZeppelin** [comment]
I have no intentions on doing that.

I may however make Divest apps available via a forked GrapheneOS Apps app.

**Reply by rsssxd** [reply]
This is not necessary. Accrescent already shares the same foundation as the GrapheneOS app store. This means users install more unnecessary app stores.

**Reply by SkewedZeppelin** [reply]
I am philosophically opposed to Accressant and will not consider publishing my apps there.

**Reply by matchboxbananasynergy** [reply]
> This is not necessary. Accrescent already shares the same foundation as the GrapheneOS app store. This means users install more unnecessary app stores.

To be clear, Accrescent is not a fork of the GrapheneOS "Apps" app, so you may be confused about that.

**Reply by grrrrr** [reply]
Hi @SkewedZeppeiln would you be able to expand on your philosophical opposition to Accresent, I'd be interested to understand more

**Reply by SkewedZeppelin** [reply]
I ain't got time to publish/maintain apps on another place.

You can get my binaries from my repo: https://divestos.org/pages/our_apps#repos
And you can F-Droid built binaries from F-Droid.org

**Reply by ghost** [reply]
> I have no intentions on doing that.
> 
> I may however make Divest apps available via a forked GrapheneOS Apps app.

That would be great!

**Reply by steadfasterX** [reply]
haha that reminds about GRM :D https://github.com/sfX-android/GRM
